src/cms/urls.py

```diff
@@ -291,6 +291,18 @@
         r"^ajax/",
         include(
             [
+                url(
+                    r"^render/",
+                    include(
+                        [
+                            url(
+                                r"^mirrored_page_field/",
+                                pages.render_mirrored_page_field,
+                                name="render_mirrored_page_field",
+                            ),
+                        ]
+                    ),
+                ),
```

Comment:
Why is it necessary to use ajax here? The only case I can imagine is, when the user opens the form and wants keep track of changes to all other regions pages without refreshing, is this right?

Reply:
Yes, it's not technically necessary, it just avoids a huge overhead when rendering the page form... otherwise the `mirrored_page_field` would have to be rendered for every city in the cms beforehand (and thus one option for every page object in the whole cms... in the actual live data, this can exceed a few thousand entries). Since the live content region is only changed rarely, we don't need to render a few thousand option tags every time a page form gets opened.

Reply:
I see, so it prevents database from being overload:thumbsup:.